### DISCUSSION TITLE
Is pointerconfig props avaliable on Bar chart?

### DISCUSSION THREAD

**Discussion by harvey-seunpil-kim** [Abhinandan-Kushwaha/react-native-gifted-charts#444] [Q&A] [answered]
Is pointerconfig props avaliable on Bar chart?

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Yes, it's available for Bar charts as well. See [pointerConfig in Bar Chart props](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/docs/BarChart/BarChartProps.md#pointerconfig)

**Reply by harvey-seunpil-kim** [reply]
Thanks for answer,
I tried to use it but I can't see pointerStrip at all.
Is there any example or some code that use pointerConfig props with (stack) bar chart?

Thanks
